src/DynamoCoreWpf/UI/Themes/Modern/Ports.xaml

```diff
@@ -114,8 +99,8 @@
                                 <Setter Property="Grid.Margin" Value="{Binding Path=MarginThickness}" />
                                 <Style.Triggers>
                                     <DataTrigger Binding="{Binding Path=IsPortCondensed}" Value="True">
-                                        <Setter Property="Grid.Height" Value="{Binding Source={x:Static configuration:Configurations.CodeBlockOutputPortHeightInPixels}}" />
```

Comment:
Can you explain about this change here?

Reply:
Yeah I changed this from being the config value to something hard-coded. The visual mockup shows the port as slightly smaller than the full port height in order to introduce a bit of a visual gap between ports.

Reply:
We still use this in a few places. There's an invisible grid that's the full line-to-line height which needs to be this exact value in the config, but the light grey rectangle we display is slightly less than this for graphics' sake.